src/components/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -29,14 +32,23 @@ export const HorizontalTabs = ({
   size = HorizontalTabSize.LG,
   sx,
   renderContent,
+  autoSelectFirst = true,
+  syncWithValue = false,
```

Comment:
This was needed to force the tabs to be unselected for the mobile tab navigation